api/catalog/settings.py

```diff
@@ -350,3 +351,9 @@
         send_default_pii=False,
         environment=ENVIRONMENT,
     )
+
+    # ALLOW_HOSTS is correctly configured so ignore this to prevent
+    # spammy bots like https://github.com/robertdavidgraham/masscan
+    # from pushing un-actionable alerts to Sentry like
+    # https://sentry.io/organizations/openverse/issues/3252810791
```

Comment:
Should this be the public URL instead? Openverse maintainers can go to the details from there.